api/src/main/java/com/redhat/insights/tls/PEMSupport.java

```diff
@@ -52,7 +52,7 @@ public SSLContext createTLSContext(Path certificatePath, Path keyPath) {
       byte[] certBytes = Files.readAllBytes(certificatePath);
       byte[] keyBytes = Files.readAllBytes(keyPath);
       return createTLSContext(certBytes, keyBytes);
-    } catch (IOException err) {
+    } catch (Throwable err) {
```

Comment:
Why Throwable and not Exception ?

Reply:
It's a habit from Mutiny where we catch `Throwable` 👍 
Fixing it